google/cloud/spanner/transaction.h

```diff
@@ -95,6 +97,23 @@ class Transaction {
     google::spanner::v1::TransactionOptions_ReadWrite rw_opts_;
   };
 
+  /**
+   * Options for Partitioned DML transactions.
+   *
+   * @see
+   * https://cloud.google.com/spanner/docs/transactions#partitioned_dml_transactions
+   *   for an overview of Partitioned DML transactions.
+   */
+  class PartitionDmlOptions {
```

Comment:
> Apart from the shape of comments, the transaction.{h,cc} changes are identical to mine.

Oh ... except that I called this `Partition*ed*DmlOptions`.

Reply:
Fixed.